lib/chef/knife/bootstrap.rb

```diff
@@ -22,229 +22,31 @@
 require "chef/knife/bootstrap/chef_vault_handler"
 require "chef/knife/bootstrap/client_builder"
 require "chef/util/path_helper"
+require "chef/knife/bootstrap/options"
 
 class Chef
   class Knife
     class Bootstrap < Knife
       include DataBagSecretOptions
+      # Command line flags and options for bootstrap - there's a large number of them
+      # so we'll keep this file a little smaller by splitting them out.
+      include Bootstrap::Options
+
+      SUPPORTED_CONNECTION_PROTOCOLS = %w{ssh winrm}.freeze
 
       attr_accessor :client_builder
       attr_accessor :chef_vault_handler
+      attr_reader   :target_host
 
       deps do
-        require "chef/knife/core/bootstrap_context"
         require "chef/json_compat"
         require "tempfile"
-        require "highline"
```

Comment:
Are we actually killing off highline entirely with this change. I know we only use it in a handful of places. If so we should nuke that dep in the gemspec.

Reply:
We actually weren't using highline in bootstrap